### DISCUSSION TITLE
TipTap 2 community repo(s)

### DISCUSSION THREAD

**Discussion by bgrand-ch** [ueberdosis/tiptap#921] [Feature Requests]
- create a repo to list all community extensions, like this : https://github.com/rollup/awesome

or/and

- create a repo to integrate all community extensions, like this : https://github.com/nativescript-community

**Comment by hanspagel** [comment]
Thanks! Both exist or are on the way:

tiptap awesome: https://github.com/ueberdosis/awesome-tiptap
Community extensions: #819 (pinned to the top)

We’ll improve both solutions with tiptap 2 and promote them more then. ✌️
